packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -151,8 +152,17 @@ - (void)interpretInfoWindow:(NSDictionary *)data {
   }
 }
 
+/// Extracts an icon image from the iconData array.
+///
+/// @param iconData An array containing the data for the icon image.
+/// @param registrar A Flutter plugin registrar.
+/// @param screenScale Screen scale factor for scaling bitmaps. Must be greater than 0.
+/// @return A UIImage object created from the icon data.
+/// @note Assert unless screenScale is greater than 0.
```

Comment:
Declaration comments shouldn't be duplicated; this comment is in the test header.